test/test.js

```diff
@@ -491,6 +491,19 @@ suite('Request', function() {
     })
   })
 
+  test('clone request bodies', function() {
+    var r1 = new Request('https://fetch.spec.whatwg.org/', {
+      method: 'post',
+      headers: {'content-type': 'text/plain'},
+      body: 'I work out'
```

Comment:
I'm unsure since I haven't touched on that part of the code but if we're not polyfilling Blob I'd say it seems a bit unnecessary to test the browser implementation. Did I understand it correctly?